azure-pipelines.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       cp $tmpdir\$(BITS)\lib\libopenblas_v0.3.5-274-g6a8b4269-gcc_7_1_0.a $target
     displayName: 'Download / Install OpenBLAS'
   - powershell: |
-      choco install -y mingw --forcex86 --force --version=5.3.0
+      choco install -y mingw --forcex86 --force --version=7.3.0
```

Comment:
Is the old version causing problems in master branch CI or something? As you likely know, the old version is used because it provides the gcc / gfortran runtime library dependencies that OpenBLAS needs. You could try to bump our upstream OpenBLAS binary builds to have more modern gcc runtime dependencies, though I wonder about manylinux support.

We could also static link upstream, but that discussion went nowhere not that long ago, and on many flavors of Linux it would require a custom rebuild of gcc itself.

Reply:
It seems the old version is no longer publically available on [sourceforge](https://sourceforge.net/projects/mingw-w64/files/). There was a CI failure that I thought was triggered by 5.3 being removed from that link, but it seems it was a temporary outage because later builds succeed.

Strangely, I was not able to reproduce this failure locally when mimicking the steps in the azure 32-bit windows build in this branch: everything succeeded.